pydoctor/templatewriter/summary.py

```diff
@@ -273,7 +273,7 @@ def roots(self, request, tag):
     def rootkind(self, request, tag):
         rootkinds = {}
         for o in self.system.rootobjects:
-            rootkinds[o.kind.lower() + 's']  = 1
+            rootkinds[epydoc2stan.format_kind(o.kind, plural=True).lower()] = 1
         return tag.clear()('/'.join(sorted(rootkinds)))
```

Comment:
I think this is ancient code from a time before Python supported generator comprehensions or even sets. In modern Python it can be written more elegantly like this:
```py
         return tag.clear()('/'.join(sorted(
             epydoc2stan.format_kind(o.kind, plural=True).lower()
             for o in self.system.rootobjects
             )))
```